start.sh

```diff
@@ -24,14 +24,14 @@ fi
 
 echo "Setting up python virtual environment."
 if [ ! -f "venv/bin/activate" ]; then
-    python3.8 -m venv "venv"
+    python3 -m venv "venv"
```

Comment:
the version is explicit is because we say we support it in pyscitt config, why drop it?